magicctapipe/scripts/lst1_magic/lst1_magic_event_coincidence.py

```diff
@@ -136,21 +185,22 @@ def event_coincidence(input_file_lst, input_dir_magic, output_dir, config):
 
     window_half_width = u.Quantity(window_half_width).to("ns")
     window_half_width = u.Quantity(window_half_width.round(), dtype=int)
-
     pre_offset_search = False
-    if "pre_offset_search" in config_coinc:
-        pre_offset_search = config_coinc["pre_offset_search"]
+    if "pre_offset_search" in config_coinc: #looking for the boolean value of pre_offset_search in the configuration file
+        pre_offset_search = config_coinc["pre_offset_search"] 
```

Comment:
you can drop the extra space here (that triggers the change of the line)

Reply:
Done